apps/qa/src/checkbook/checkbook.py

```diff
@@ -2,21 +2,10 @@ def checkbook():
     import streamlit as st
     from src.publishing import read_csv_cached
     from src.components import sidebar
+    from src.components.get_data_directory_url import get_data_directory_url
```

Comment:
final final nit, sorry - I'd like to change this function name. Just ideally, a component name should represent its entity. So instead of a sort description of an action, which is a relevant name for a helper function, a description of the component itself - `sidebar.selectbox`, etc. So something like `data_directory_link()`